### PR TITLE
Add Toad.app v2.3.6

### PR DESCRIPTION
**Pull request by mochadwi**
Add Toad - Mac Community Edition database management tools for OS X, Support Oracle Database, MySQL, PostgreSQL and MongoDB. It's freeware and awesome xD

### CODE REVIEW

File: Casks/toad.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask 'toad' do
+  version '236'
```
Comment (adidalal) [inline Casks/toad.rb]: 2.3.6

File: Casks/toad.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+  sha256 '012b866e16a33293dc3ff73a4864f5ae19213b7e6f3032b37df1f712a2c0f898'
+
+  url "http://community-downloads.quest.com/toadsoft/toadmacedition/ToadMacEdition_#{version}.zip"
+  name 'Toad – Mac Edition Community'
```
Comment (adidalal) [inline Casks/toad.rb]: Toad

File: Casks/toad.rb [hunk 3]
```diff
@@ -0,0 +1,11 @@
+cask 'toad' do
+  version '236'
+  sha256 '012b866e16a33293dc3ff73a4864f5ae19213b7e6f3032b37df1f712a2c0f898'
+
+  url "http://community-downloads.quest.com/toadsoft/toadmacedition/ToadMacEdition_#{version}.zip"
```
Comment (adidalal) [inline Casks/toad.rb]: use version.no_dots

Comment (adidalal) [inline Casks/toad.rb]: needs URL comment https://github.com/caskroom/homebrew-cask/pull/18051

**Comment by vitorgalvao** [comment]
Thank you for the contribution. It needed some fixes, so they were made in 7d0b3a45db2688a79b7b8e2e4cfb656bbeb997fe. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

**Comment by mochadwi** [comment]
Hello @vitorgalvao thanks! I'll merge my commit with yours.

**Comment by adidalal** [comment]
Changes already made, you can just brew update once you switch back to master branch, if needed

**Comment by mochadwi** [comment]
Okay, thanks @adidalal !

**Comment by mochadwi** [comment]
Okay, thanks @adidalal !

**Pull request closed without merging**